Documentation/common-problems.md

```diff
@@ -246,25 +247,55 @@ What is happening here is that the MON pods are restarting and one or more of th
 
 The `dataDirHostPath` setting specifies a path on the local host for the CEPH daemons to store configuration and data. Setting this to a path like `/var/lib/rook`, reapplying your Cluster CRD and restarting all the CEPH daemons (MON, MGR, OSD, RGW) should solve this problem. After the CEPH daemons have been restarted, it is advisable to restart the rook-api and [rook-tool Pods](./toolbox.md).
 
-# Only a single monitor pod starts after redeploy
+# Only a single monitor pod starts
```

Comment:
so this can also happen during first setup and not just redeploy?

Reply:
Some error conditions in this topic can happen on a new install, and some on a redeploy.